cfgov/v1/blocks.py

```diff
@@ -198,106 +196,3 @@ class PlaceholderCharBlock(PlaceholderFieldBlock, blocks.CharBlock):
 class ReusableTextChooserBlock(SnippetChooserBlock):
     class Meta:
         template = '_includes/snippets/reusable_text.html'
-
-
-class Link(blocks.StructBlock):
```

Comment:
I'm fine with getting rid of this, because there's probably a better way to do it, but here's yet another reminder that Wagtail needs a LinkChooserBlock.